### ISSUE TITLE
Conflicts with stand-alone MMD Tools, which is more up to date. Way to update?

### ISSUE THREAD

**Issue by Jatts-Art** [absolute-quantum/cats-blender-plugin#395]
Honestly, wouldnt it be better to have MMD Tools be optional? Because, as the title says, it conflicts with MMD Tools (as in, some tabs are overrided) if added as a separate addon but that one is more up to date than this version's. As for CATS, I only really needed it so I can apply poses as a rest post, to begin with. Its not a major issue of course, but figured I'd mention this anyway in case theres a way I can swap the MMD Tools folder inside or something. Thanks!

**Comment by 989onan** [comment]
Yeah there is a way but it's very involved if you want to do it manually, 
but the jist is to find your blender addons location, take the mmd_tools folder content and then put it inside Cat's MMD folder inside the cats plugin folder. I do not think it will work it will probably crash or error due to cats depending on how the old version was coded to work and unexpected changes in the new version might just brick cats

**Comment by Jatts-Art** [comment]
Sadly that is actually one of the first things I had done before making this thread, haha... and yeah it didnt work as many of the tabs were completely missing.

**Comment by Darkblader24** [comment]
There were so many people having issues trying to install mmd_tools, so we just included it. I regularily update mmd_tools for each release and recently updated it in the Dev version of CATS. I saw there was another update to mmd_tools so I will update it again soon. Also in CATS there is a setting to hide mmd_tools, you can try unchecking that.

**Comment by Jatts-Art** [comment]
I unchecked that and yeah same issue, it would disable tabs relating to my other install of MMD Tools even if its outside the folder. But still i appreciate the responses either way, I'll just deal with having both installed at once and try to remember which tabs are what I suppose lol
